src/nls/fi/strings.js

```diff
@@ -624,7 +624,25 @@ define({
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Viimeisimmät projektit",
 
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Lue lisää"
+    "DOCS_MORE_LINK"                            : "Lue lisää",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Pienennä kaikki",
```

Comment:
What does collapse custom regions do?
Also putting in all the code folding options in the top level view menu would increase the menu real estate. It would be ideal to move it into a top level 'code folding' entry in the view/edit menu(as in sublime ) and then put the folding options as submenus. The string freeze is by 6'th, so we would need to put in all strings into brackets before that.